src/library/basetracktablemodel.cpp

```diff
@@ -474,6 +478,37 @@ QVariant BaseTrackTableModel::roleValue(
     case Qt::ToolTipRole:
         if (column == fieldIndex(ColumnCache::COLUMN_LIBRARYTABLE_COLOR)) {
             return mixxx::RgbColor::toQString(mixxx::RgbColor::fromQVariant(rawValue));
+        } else if (column == fieldIndex(ColumnCache::COLUMN_LIBRARYTABLE_COVERART)) {
+            // Determine height of the cover art image depending on the screen size
```

Comment:
Readability: Extract this code into one or more functions, either member or free functions in an anonymous namespace.

Reply:
Done